### PR TITLE
method for notifying content view to set webcontents

### PR DESCRIPTION
**Pull request by deepak1556**

### CODE REVIEW

File: browser/inspectable_web_contents.h [hunk 1]
```diff
@@ -35,6 +35,7 @@ class InspectableWebContents {
   virtual InspectableWebContentsDelegate* GetDelegate() const = 0;
 
   virtual void SetCanDock(bool can_dock) = 0;
+  virtual void SetInspectedWebContents() = 0;
```
Comment (zcbenz) [inline browser/inspectable_web_contents.h]: We don't need to add this method here, we can just call `GetView()->SetInspectedWebContents()`

File: browser/inspectable_web_contents_view.h [hunk 2]
```diff
@@ -40,6 +40,7 @@ class InspectableWebContentsView {
   virtual gfx::NativeView GetNativeView() const = 0;
 #endif
 
+  virtual void SetInspectedWebContents() = 0;
```
Comment (zcbenz) [inline browser/inspectable_web_contents_view.h]: `SetInspectedWebContents` is a quite confusing name, probably use `InstallWebContentsView` instead?

Comment (zcbenz) [inline browser/inspectable_web_contents_view.h]: Also can you add an empty definition for OS X?

Comment (deepak1556) [inline browser/inspectable_web_contents_view.h]: FIxed, thanks!

**Comment by deepak1556** [comment]
Ref https://github.com/atom/electron/pull/3168#issuecomment-149924983

**Pull request closed without merging**